Signal/src/ViewControllers/DebugUI/DebugUISessionState.m

```diff
@@ -49,13 +49,20 @@ + (OWSTableSection *)sectionForContactThread:(TSContactThread *)contactThread
                                        actionBlock:^{
                                            [self presentVerificationStatePickerForContactThread:contactThread];
                                        }],
-                       [OWSTableItem itemWithTitle:@"Delete session"
+                       [OWSTableItem itemWithTitle:@"Delete all sessions"
                                        actionBlock:^{
                                            dispatch_async([OWSDispatch sessionStoreQueue], ^{
                                                [[TSStorageManager sharedManager]
                                                    deleteAllSessionsForContact:contactThread.contactIdentifier];
```

Comment:
If we're only going to use deleteAllSessionsForContact for debugging perhaps we can `#ifdef DEBUG`?